src/tests/ov_helpers/ov_snippets_models/include/subgraph_mlp.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets_helpers.hpp"
+
+/* The file contains graphs with different MLP-patterns:
+ * Skeleton of MLP-pattern is:
+ *       Input
+ *         |
+ *      MatMul
+ *         |
+ *      Eltwise
+ *         |
+ *      MatMul
+ *         |
+ *      Output
+ */
+
+namespace ov::test::snippets {
+
+/* Graph:
+ *       Input
+ *         |
+ *        FQ
```

Comment:
This graph is related to `MLPSeqQuantizedTypeRelaxedFunction`, right?  Can we move it to the corresponding class then? To avoid misunderstandings